test/org/zaproxy/zap/extension/openapi/HTTPDTestServer.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Well if the issue goes away when we do away with the current branching model, then lets not waste additional time on it at this point.

Do we have the same problem between the main project and extensions? If so then perhaps it's something we should tackle from that perspective.